test/tests/transfers/general_field/user_object/nearest_position/main_between_multiapp.i

```diff
@@ -28,20 +28,24 @@
 []
 
 # This application uses at most 3 processes
-[MultiApps/ma1]
-  type = TransientMultiApp
-  input_files = sub_between_diffusion.i
-  max_procs_per_app = 3
-  positions_objects = 'input_app1'
-  output_in_position = true
+[MultiApps]
+  [ma1]
+    type = TransientMultiApp
+    input_files = sub_between_diffusion.i
+    max_procs_per_app = 3
+    positions_objects = 'input_app1'
+    output_in_position = true
+  []
```

Comment:
k+s. I don't know how many times I have to say it. I just don't think git blame should point to a commit in this PR

Reply:
yeah that does not work. I ll just remap it to something that works if you care

Reply:
Cmd k + s to be more explicit
<img width="1150" alt="Screenshot 2025-05-29 at 3 00 31 PM" src="https://github.com/user-attachments/assets/892daf87-9f46-48e3-bc68-1027e9e3193c" />

I've brought this up on your PRs at least a half dozen times. I wouldn't keep saying it if I didn't care. Every time I edit an input file I save without formatting to reduce the diff. It makes it easier for a reviewer to see the meaningful changes and it makes `git blame` more helpful

Reply:
do you press cmd K then quickly s? the three at the same time? Does not do it on my system. It actually counts it as cmd s + cmd k and pulls up this list of all commands.

Reply:
cmd K, release fingers off keys, then s